frontend/src/metabase/visualizations/components/settings/ChartSettingsTableFormatting.jsx

```diff
@@ -92,106 +91,131 @@ const INPUT_CLASSNAME = "mt1 full";
 const getValueForDescription = rule =>
   ["is-null", "not-null"].includes(rule.operator) ? "" : ` ${rule.value}`;
 
-export default class ChartSettingsTableFormatting extends Component {
-  state = {
-    editingRule: null,
-    editingRuleIsNew: null,
-  };
+const ChartSettingsTableFormatting = props => {
```

Comment:
Can we update the exports and imports to use non-default imports?

```suggestion
export function ChartSettingsTableFormatting(props) {
```

Reply:
👍